enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -40,6 +40,7 @@
 socket_timeout = float(os.getenv('EG_SOCKET_TIMEOUT', '5.0'))
 tunneling_enabled = bool(os.getenv('EG_ENABLE_TUNNELING', 'False').lower() == 'true')
 ssh_port = int(os.getenv('EG_SSH_PORT', '22'))
+advertised_host = os.getenv('EG_ADVERTISED_HOST_LISTENER', None)
```

Comment:
Please change the env variable name to `EG_RESPONSE_IP` and add an entry about it to https://github.com/jupyter/enterprise_gateway/blob/master/docs/source/config-options.md#addtional-supported-environment-variables.  
Please mention this should be used when kernels reside in a different logical network than the EG container or pod.

It would also be helpful to describe how the appropriate IP address is determined - public IP, etc?